libs/@hashintel/design-system/package.json

```diff
@@ -1,17 +1,18 @@
 {
   "name": "@hashintel/design-system",
-  "version": "0.0.0-private",
-  "private": true,
+  "version": "0.0.0",
   "description": "HASH Design System",
   "license": "AGPL-3.0",
   "sideEffects": false,
-  "main": "./src/main.ts",
-  "types": "./src/main.ts",
+  "main": "src/main.ts",
+  "types": "src/main.ts",
```

Comment:
This change makes paths shorter and closer to docs: https://docs.npmjs.com/cli/v9/configuring-npm/package-json